toolchain/pkg.yaml

```diff
@@ -9,6 +9,19 @@ dependencies:
   - stage: make
   - stage: golang
 steps:
+  - build:
```

Comment:
shouldn't this be the test stage?

Reply:
No, because I also do actual symlinks (stagex doesn't provide ones)